.github/workflows/qc.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           pip install flake8 flake8-black flake8-isort
       - name: Check code quality 
         run: |
-          flake8 --ignore F401 src/ test/
+          flake8 --ignore=F401,E203 src/ test/
```

Comment:
Remember to remove the ignore for the E203.

Reply:
https://github.com/INCATools/PandaSaurus/issues/15 If we go with black convention we have to keep it. Otherwise it will fail the qc... If I don't ignore it and remove the space before `:` a black warning will fail the qc

Reply:
You already added the space before `:`, so you no longer need to ignore that.

Reply:
There are two different warning/error related with space before `:`. One is E203 which requires you to 'REMOVE' the space in order to resolve it, and the other is 'Black would make changes' which you can resolve by 'ADDING' the space. We HAVE to keep the ignore statement in oder to PASS the qc.